<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VariablePlotDialog</class>
 <widget class="QDialog" name="VariablePlotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Variable Plot Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="2" column="0" colspan="4">
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="cancelButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="okButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="projectgroupBox">
     <property name="minimumSize">
      <size>
       <width>190</width>
       <height>220</height>
      </size>
     </property>
     <property name="title">
      <string>Project List</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QListView" name="projectView">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QGroupBox" name="datagroupBox">
     <property name="minimumSize">
      <size>
       <width>190</width>
       <height>220</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Data</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QListView" name="dataView">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QGroupBox" name="objectgroupBox">
     <property name="minimumSize">
      <size>
       <width>190</width>
       <height>220</height>
      </size>
     </property>
     <property name="title">
      <string>Objects</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QListView" name="objectsView">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="variablegroupBox">
     <property name="minimumSize">
      <size>
       <width>190</width>
       <height>190</height>
      </size>
     </property>
     <property name="title">
      <string>Variable</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QListView" name="variableView">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QGroupBox" name="variablegroupBox_2">
     <property name="minimumSize">
      <size>
       <width>190</width>
       <height>190</height>
      </size>
     </property>
     <property name="title">
      <string>Variable</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QListView" name="variableView_2">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
